### PR TITLE
Home task: HTML 01

### PR DESCRIPTION
**Pull request by YauheniyaNem**
Could you check my homework HTML.01
Sorry for it`s so late.
Thanks

### CODE REVIEW

File: enemytskova/01.HTML/Task3.html [hunk 1]
```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html> 
+<HTML> 
+    <HEAD> 
+        <META charset="utf-8"> 
+        <TITLE>Оформление текста, Python, Часть1</TITLE> 
+    </HEAD> 
+    <BODY> 
+        <h1>HOW-TO: Программа на Python, Часть 1</h1> 
+        <ul>
+            <i>
```
Comment (pluhin) [inline enemytskova/01.HTML/Task3.html]: inline tag sould be inside each li tag

File: enemytskova/01.HTML/Task3.html [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    	    <li>Программа на Python — часть 1</li>
+    	    <li>Программа на Python — часть 2</li>
+    	    <li>Программа на Python — часть 3</li>
+            </i>
+    	</ul>   
+    	<h2>Содержание</h2>
+    	<h3>HOW-TO: Программа на Python, Часть 1</h3>
+    	<ul>
+            <u>
```
Comment (pluhin) [inline enemytskova/01.HTML/Task3.html]: inline tag sould be inside each li tag

File: enemytskova/01.HTML/Task4.html [hunk 3]
```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html> 
+<HTML> 
+    <HEAD> 
+        <META charset="utf-8"> 
+        <TITLE>Python, Часть 2</TITLE> 
+    </HEAD> 
+    <BODY> 
+        <h1>HOW-TO: Программа на Python, Часть 2</h1> 
+        <ul>
+            <i>
```
Comment (pluhin) [inline enemytskova/01.HTML/Task4.html]: inline tag sould be inside each li tag

File: enemytskova/01.HTML/Task4.html [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+    	    <li>Программа на Python — часть 1</li>
+    	    <li>Программа на Python — часть 2</li>
+    	    <li>Программа на Python — часть 3</li>
+            </i>
+    	</ul>   
+    	<h2>Содержание</h2>
+    	<h3>HOW-TO: Программа на Python, Часть 2</h3>
+    	<ul style="margin-bottom: 1.5em">
+            <u>
```
Comment (pluhin) [inline enemytskova/01.HTML/Task4.html]: inline tag sould be inside each li tag

File: enemytskova/01.HTML/Task4.html [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+        списков.</p>
+        <hr>
+        <h2>Комментарии</h2>
+        <p>Далее, мы должны обсудить комментарии. Комментарии важны по нескольким причинам.<br>
+        Они не только говорят вам или другим людям, что вы пытаетесь делать, но и напомнят<br>
+        вам через полгода, что же вы хотели сделать. Когда вы будете писать много программ,<br>
+        комментарии станут важными. Также при их помощи вы заставите Python игнорировать<br>
+        какие-либо части кода. Чтобы закомментировать строку, поставьте перед ней знак '#'.<br>
+        Например:</p>
+        #
```
Comment (pluhin) [inline enemytskova/01.HTML/Task4.html]: it is code too

File: enemytskova/01.HTML/Task5.html [hunk 6]
```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html> 
+<HTML> 
+    <HEAD> 
+        <META charset="utf-8"> 
+        <TITLE>Python, Часть 3</TITLE> 
+    </HEAD> 
+    <BODY> 
+        <h1>HOW-TO: Программа на Python, Часть 3</h1> 
+        <ul>
+            <i>
```
Comment (pluhin) [inline enemytskova/01.HTML/Task5.html]: inline tag sould be inside each li tag

File: enemytskova/01.HTML/Task5.html [hunk 7]
```diff
@@ -0,0 +1,101 @@
...
+    	    <li>Программа на Python — часть 1</li>
+    	    <li>Программа на Python — часть 2</li>
+    	    <li>Программа на Python — часть 3</li>
+            </i>
+    	</ul>   
+    	<h2>Содержание</h2>
+    	<h3>HOW-TO: Программа на Python, Часть 3</h3>
+    	<ul style="margin-bottom: 1.5em">
+            <u>
```
Comment (pluhin) [inline enemytskova/01.HTML/Task5.html]: inline tag sould be inside each li tag

**Review by pluhin — CHANGES_REQUESTED** [review]

File: enemytskova/01.HTML/Task4.html (hunk 5 above)
Comment (YauheniyaNem) [reply enemytskova/01.HTML/Task4.html]: Good evening! Sorry, I dont`t understand what do you mean) Could you explaine me, please

**Review by pluhin — APPROVED** [review]
good, approved

**Comment by YauheniyaNem** [comment]
Thanks

**Pull request merged**